sfft_and_animate.py

```diff
@@ -1,300 +1,308 @@
-"""
-Saves a gif and does SFFT on the specified SN. 
-
-"""
-
 # IMPORTS Standard:
+import argparse
 import os
 import sys
 import matplotlib.pyplot as plt
 from mpl_toolkits.axes_grid1 import make_axes_locatable
 import numpy as np
-from copy import copy
 
 # IMPORTS Astro:
 from astropy.table import Table
 from astropy.coordinates import SkyCoord
 from astropy.visualization import ZScaleInterval
 from astropy.io import fits
 from astropy.nddata import Cutout2D
+from astropy.wcs import WCS
 from astropy.nddata.utils import PartialOverlapError, NoOverlapError
 import astropy.units as u
 
 # IMPORTS Internal:
-from phrosty.imagesubtraction import *
+from phrosty.imagesubtraction import sky_subtract, imalign, get_imsim_psf, rotate_psf, crossconvolve, difference, decorr_kernel, decorr_img, stampmaker
 from phrosty.plotting import animate_stamps
 from phrosty.utils import get_transient_radec, get_transient_mjd, get_mjd_info, _build_filepath
 
 ###########################################################################
 
-# SLURM
-sys.path.append(os.getcwd())
-taskID=int(os.environ['SLURM_ARRAY_TASK_ID'])
-
-print('taskID', taskID)
-
-# config = {1: 20172117, 
-#           2: 20172199, 
-#           3: 20172328, 
-#           4: 20172782, 
-#           5: 20173301, 
-#           6: 20173305, 
-#           7: 20173373, 
-#           8: 20174023, 
-#           9: 20174108, 
-#           10: 20174118, 
-#           11: 20174213, 
-#           12: 20174370, 
-#           13: 20174542, 
-#           14: 20175077, 
-#           15: 20175568, 
-#           16: 20177380,
-#           17: 20202893}
-
-# oid = config[taskID]
-# bands = get_roman_bands()
-
-config = {1: 'F184',
-          2: 'H158',
-          3: 'J129',
-          4: 'K213',
-          5: 'R062',
-          6: 'Y106',
-          7: 'Z087'}
-
-band = config[taskID]
-print(band)
-overlap_size=1000
-
-###########################################################################
-
-def sfft_and_animate(oid,band):
+def set_sn(oid):
```

Comment:
Would `get_sn_info` be another good name for this function?

Reply:
Yeah probably